include/EclipseMonitor/Trie/BranchNode.hpp

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 Tuan Tran
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+#pragma once
+
+#include "EclipseMonitor/EthKeccak256.hpp"
```

Comment:
Use relative paths (i.e., the path that is relative to the current file) instead of absolute paths, when referencing the file within the same project. So this will be `#include "../EthKeccak256.hpp"`